graylog2-server/src/main/java/org/graylog2/rest/resources/search/SearchResource.java

```diff
@@ -112,7 +112,7 @@ protected void validateInterval(String interval) {
         } catch (Searches.FieldTypeException e) {
             try {
                 LOG.debug("Stats query failed, make sure that field [{}] is a numeric type. Retrying without numeric statistics to calculate the field's cardinality.", field);
-                return searches.fieldStats(field, query, filter, timeRange, true, true);
+                return searches.fieldStats(field, query, filter, timeRange, true, false, false);
```

Comment:
I think we should be calling this with a `true` in the include count argument:
`return searches.fieldStats(field, query, filter, timeRange, true, false, true);`